flake.nix

```diff
@@ -12,15 +13,17 @@
   outputs = {
     self,
     nixpkgs,
+    php-nixpkgs,
     flake-utils,
     treefmt-nix,
     ...
   }: let
     buildOutputs = system: let
       pkgs = nixpkgs.legacyPackages.${system};
+      php-pkgs = php-nixpkgs.legacyPackages.${system};
       treefmtEval = treefmt-nix.lib.evalModule pkgs ./treefmt.nix;
     in {
-      packages = import ./all-packages.nix {inherit (pkgs) newScope;};
+      packages = import ./all-packages.nix {inherit (php-pkgs) newScope;};
```

Comment:
we probably want to figure out a way to use `php-pkgs` just for flarum and `pkgs` for everything else 🤔